setup.py

```diff
@@ -200,6 +203,35 @@ def run(self):
         self.build_mo()
 
 
+class PootleUpdateReadme(Command):
+    """Process a README file for branching"""
+
+    user_options = [
+        ('write', 'w', 'Overwrite the %s file' % README_FILE),
+        ('branch', 'b', 'Rewrite using branch rewriting (default)'),
+        ('release', 'r', 'Rewrite using release or tag rewriting (default)'),
```

Comment:
How can both be default? I mean, below it looks like specifying one of them alters the other (despite not being used?)...

Is it possible to simplify these options?